cpp-unisocket/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM fedora:34
+
+ARG VERSION=4.2
```

Comment:
Building Docker image failed for me with the following error.

```
Complete!
Removing intermediate container 93d8b150968c
 ---> 29834c10177a
Step 4/8 : RUN wget --quiet https://github.com/hazelcast/hazelcast-cpp-client/archive/v${VERSION}.zip && unzip -q v${VERSION}.zip && rm v${VERSION}.zip && cd hazelcast-cpp-client-${VERSION} && cmake -S . -B build && cmake --build build --verbose && cmake --install build && cd .. && rm -rf hazelcast-cpp-client-${VERSION}
 ---> Running in 36c9ac2ed9f1
The command '/bin/sh -c wget --quiet https://github.com/hazelcast/hazelcast-cpp-client/archive/v${VERSION}.zip && unzip -q v${VERSION}.zip && rm v${VERSION}.zip && cd hazelcast-cpp-client-${VERSION} && cmake -S . -B build && cmake --build build --verbose && cmake --install build && cd .. && rm -rf hazelcast-cpp-client-${VERSION}' returned a non-zero code: 8
```

Any thoughts?

Reply:
Because we did not release 4.2 version yet. Hence, you can change `v${VERSION}` with `master` branch and try the dockerfile.

Reply:
ahh, ok. Now it works :)